src/frontend/src/lib/derived/network-tokens.derived.ts

```diff
@@ -6,23 +6,36 @@ import { tokens, tokensToPin } from '$lib/derived/tokens.derived';
 import { balancesStore } from '$lib/stores/balances.store';
 import type { Token, TokenUi } from '$lib/types/token';
 import { filterTokensForSelectedNetwork } from '$lib/utils/network.utils';
-import { pinTokensWithBalanceAtTop, sortTokens } from '$lib/utils/tokens.utils';
+import {
+	filterEnabledNetworkTokens,
+	pinTokensWithBalanceAtTop,
+	sortTokens
+} from '$lib/utils/tokens.utils';
 import { derived, type Readable } from 'svelte/store';
 
 /**
  * All tokens matching the selected network or chain fusion, regardless if they are enabled by the user or not.
  */
-const networkTokens: Readable<Token[]> = derived(
+const selectedNetworkTokens: Readable<Token[]> = derived(
```

Comment:
may I ask, why renaming it? I think the old name still fits the scope, no? (and less verbose?)